### PR TITLE
add azure deployment key

### PR DESCRIPTION
**Pull request by charmilahruthika**

### CODE REVIEW

**Comment by sameersegal** [comment]
Where is the value getting set for `azure_deployment`? It should be read from `env`.

Also, let's stick to `azure_deployment_name` to be consistent with Azure documentation

**Comment by shreypandey** [comment]
Please update the example provided in quickstart tutorial along the steps in quickstart tutorial.

**Comment by charmilahruthika** [comment]
Renamed `azure_deployment` to `azure_deployment_name` and updated the example, steps and image of quickstart.

File: docs/tutorials/quickstart.md [hunk 1]
```diff
@@ -99,10 +99,10 @@ The detailed information about the fields are given below:
     1. **Name [Mandatory]** is the name of the bot. It should be the name of class for your bot code mentioned below. For this example, use `CarWashDealerFSM`.
     2. **Code [Mandatory]** is the fsm.py file python code. Copy the contents of [python file](car_wash.py) and paste it.
     3. **version [Mandatory]** - version of the bot. Put `1.0.0`.
-    4. **required_credentials [Mandatory]** - Credentials required by the bot to access various external services. Here the FSM depends on `AZURE_OPENAI_API_KEY`, `AZURE_OPENAI_API_VERSION` and `AZURE_OPENAI_API_ENDPOINT`, so put these keys in this section seperated by comma.
+    4. **required_credentials [Mandatory]** - Credentials required by the bot to access various external services. Here the FSM depends on `AZURE_OPENAI_API_KEY`, `AZURE_OPENAI_API_VERSION`, `AZURE_OPENAI_API_ENDPOINT` and `AZURE_DEPLOYMENT_NAME` so put these keys in this section seperated by comma.
```
Comment (shreypandey) [inline docs/tutorials/quickstart.md:102]: Can you please remove the and so that it be copied and pasted directly.

**Comment by sameersegal** [comment]
Closing this as we fixed it in another branch

**Pull request closed without merging**